web/src/components/shared/modals/HelmDeployModal.tsx

```diff
@@ -15,7 +15,7 @@ function makeDeployCommand({
 }: {
   appSlug: string;
   chartPath: string;
-  revision: string | null;
+  revision: number | null;
   showDownloadValues: boolean;
```

Comment:
super confused as to if this is string or number but it errors out if it's a string.

Reply:
we'll clean it up later. i think we'll clean up the response types better once we've done the initial typing